test/types/string.js

```diff
@@ -561,7 +551,8 @@ describe('string', () => {
             Helper.validate(schema, [
                 ['joe@example.com', true],
                 ['joe@www.example.com', true],
-                ['joe@localhost', false, null, '"value" must be a valid email'],
```

Comment:
I believe so, extrapolating from the resolution of https://github.com/hapijs/isemail/issues/154